pkg/schema/parser_test.go

```diff
@@ -113,6 +114,30 @@ func TestParseArray(t *testing.T) {
 	}
 }
 
+func slicesEqualStr(a, b []string) bool {
```

Comment:
https://pkg.go.dev/slices

This is now in the std library as of 1.21. If you don't want to maintain this and instead implement the EqualFunc

Reply:
DIdn't notice this was a test file lol